File: ch.jnetwork.jng-controls/projects/jngcontrols-common/src/components/browser/models/browserfilesaverequest.ts
/**
 * Request zum speichern einer hochgeladenen Datei
 */
export class BrowserFileSaveRequest {
  /**
   * Constructor
   * @param init Initale Parameter
   */
  constructor(init?: Partial<BrowserFileSaveRequest>) {
    Object.assign(this, init);
  }

  /**
   * Pfad in welchem der Upload gespeichert werden soll
   */
  public Path: string;
  /**
   * Upload ID
   */
  public UploadId: string;
  /**
   * Erlaubte Dateitypen
   */
  public AllowedTypes: string;
}

File: ch.jnetwork.sac-backend/Api/Model/Browser/BrowserFileSaveRequest.cs
namespace SimpleAngularControls.Api.Model.Browser
{
    /// <summary>
    /// Request zum Speichern einer Hochgeladenen Datei
    /// </summary>
    public class BrowserFileSaveRequest : IBrowserFileSaveRequest
    {
        /// <summary>
        /// Pfad in welchem die Datei gespeichert wird
        /// </summary>
        public string Path { get; set; } = string.Empty;

        /// <summary>
        /// Upload ID
        /// </summary>
        public string UploadId { get; set; } = string.Empty;

        /// <summary>
        /// Filter nach Datei Erweiterung
        /// </summary>
        public string AllowedTypes { get; set; } = string.Empty;
    }
}